module/Education/view/education/education/course.phtml

```diff
@@ -40,6 +40,15 @@ foreach ($documents as $document) {
             </div>
         </div>
         <hr>
+        <?php if(!$this->acl('education_service_acl')->isAllowed('course_document', 'download')) :?>
+        <div class="row">
+            <p><?= $this->translate('You are not allowed to view these documents!') ?></p>
+            <p><?=sprintf(
+                $this->translate('If you want to view these documents, you can either <a href=%s> login </a> or access these documents on the campus.'),
```

Comment:
Please ask @tomudding about whether you need to include translations as well; I think this is possible now